models/repo.go

```diff
@@ -792,7 +793,11 @@ func (repo *Repository) cloneLink(isWiki bool) *CloneLink {
 	if setting.SSH.Port != 22 {
```

Comment:
Not entirely correct tho, this would produce ssh://user@server:22/path URL when the standard port is used